### PR TITLE
Reorganization of proctored exam section

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
## [DOC-3635](https://openedx.atlassian.net/browse/DOC-3635)

I moved the proctored exam section out from under Offering Academic Course Credit in the course authors guide, and broke the single long page into five pages.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Doc team review (copy edit, or dev edit?): @edx/doc

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [x]  RTD draft for your branch: [http://draft-proctored-exam-reorg-201704.readthedocs.io/en/latest/course_features/proctored_exams/index.html](http://draft-proctored-exam-reorg-201704.readthedocs.io/en/latest/course_features/proctored_exams/index.html)

### Post-review

- [ ] Squash commits

### CODE REVIEW

File: en_us/course_authors/source/course_features/proctored_exams/index.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.. _CA_ProctoredExams:
+
+##########################
+Proctored Exams
+##########################
+
+This section describes how to create and manage proctored exams in Studio. If
+your course offers academic credit, you can add proctored exams to the course.
```
Comment (catong) [inline en_us/course_authors/source/course_features/proctored_exams/index.rst]: @grantgoodmanedX Strike the 2nd sentence re "If your course offers academic credit..."

**Comment by catong** [comment]
@grantgoodmanedX Looks great! I feel that there's no visibility into the parallel subsections, though: the side TOC only shows the top level section, and when you're reading one of the subsections you have no awareness of the other subsections at the same level (e.g. 12.5.1 with 12.5.2, 12.5.3, etc.) 
Could we add some cross-reference links, perhaps in the introductory sections of each topic? e.g. "For information about...see "

File: en_us/course_authors/source/course_features/proctored_exams/index.rst (hunk 1 above)
Comment (grantgoodmanedX) [reply en_us/course_authors/source/course_features/proctored_exams/index.rst]: Done.

**Comment by grantgoodmanedX** [comment]
I added some cross-references in the head paragraph of each page (except the overview, which has cross-references larded throughout).

**Comment by catong** [comment]
👍

**Pull request merged**